improver_tests/calibration/dz_rescaling/test_apply_dz_rescaling.py

```diff
@@ -121,7 +121,7 @@ def _create_scaling_factor_cube(
         Scaling factor cube.
     """
     cubelist = iris.cube.CubeList()
-    for ref_hour in [3, 9]:
+    for ref_hour in [3, 12]:
```

Comment:
I think the docstring needs updating to reflect this change

Reply:
Done.